src/app/clusters/application-basic-server/application-basic-server.cpp

```diff
@@ -40,14 +40,21 @@ using namespace chip::app::Clusters::ApplicationBasic;
 using namespace chip::AppPlatform;
 #endif // CHIP_DEVICE_CONFIG_APP_PLATFORM_ENABLED
 
+#ifdef CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT
```

Comment:
I would expect CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT to always be defined (possibly to 0).  So it's probably fine to just add its value to EMBER_AF_APPLICATION_BASIC_CLUSTER_SERVER_ENDPOINT_COUNT without the ifdef...

Reply:
Removed this ifdef